.github/workflows/windows.yml

```diff
@@ -0,0 +1,23 @@
+name: Windows GitHub CI
+
+on: [pull_request, push]
+
+jobs:
+  minimal:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
+    - name: configbuildandtest
+      env:
+        CONFIG_OPTS: no-bulk no-deprecated no-fips no-asm -DOPENSSL_SMALL_FOOTPRINT
+        TARGET: VC-WIN64A
+      run: |
+        New-Item -Path . -Name "_build" -ItemType "directory"
+        Set-Location "_build"
```

Comment:
I didn't know that we have powershell access like that. That being said, what's wrong with `mkdir` and `cd`?

Reply:
It says it is a powershell prompt so I just used those commands.. I was having some unrelated issues - will change it back when it works end to end.
Do I need to do the "nmake build_all_generated" ??
Also the tests abruptly failed and I have no idea why yet - does that cmd /c line look ok to you? maybe it timed out??

Reply:
> Do I need to do the "nmake build_all_generated" ??

Not really...  we should maintain a test of that *somewhere* though.  But that should be a separate CI job.
However, when you remove this, remember to change `nmake PERL=no-perl` to just `nmake`

Reply:
> Also the tests abruptly failed and I have no idea why yet - does that cmd /c line look ok to you? maybe it timed out??

That line looks perfectly fine, and it should be perfectly safe, I hope, to just drop `cmd /c`...  (its presence in appveyor.yml has its explanation, here's not the place to dive into that, though)